src/build.cc

```diff
@@ -138,7 +138,10 @@ void BuildStatus::BuildEdgeFinished(Edge* edge,
          o != edge->outputs_.end(); ++o)
       outputs += (*o)->path() + " ";
 
-    printer_.PrintOnNewLine("FAILED: " + outputs + "\n");
+    if (printer_.supports_color())
+        printer_.PrintOnNewLine("\x1B[31m" "FAILED: " "\x1B[0m" + outputs + "\n");
```

Comment:
Perhaps a braced ```if``` might be better please.

Reply:
@shahzadlone My preference is for braced `if`/`else`, but I am sticking to the style of nearby code.